endpoints.tf

```diff
@@ -38,6 +38,7 @@ resource "aws_vpc_endpoint" "remote_metastores" {
   service_name       = var.remote_metastores[count.index].endpoint
   subnet_ids         = split(",", lookup(var.remote_metastores[count.index], "subnets", join(",", var.subnets)))
   security_group_ids = [aws_security_group.endpoint_sg.id]
+  tags               = merge(map("Name", "${var.remote_metastores[count.index].prefix}_metastore"), var.tags)
```

Comment:
Same question here as on Apiary-Data-Lake PR: Why do we need to hardcode `Name` tag here when we have the variable `tags` which can contain `Name` as one of the tags passed from the caller of this module ?

Reply:
Got the answer in the other PR. So, approving !